docs/configuration-reference/platforms/baremetal.md

```diff
@@ -190,7 +190,8 @@ os_version = var.custom_default_os_version
 | `ignore_x509_cn_check`            | Ignore check of common name in x509 certificates. If any application is built pre golang 1.15 then API server rejects x509 from such application, enable this to get around apiserver.                                                                                                                                                                                                    | false                  | bool              | false    |
 | `install_to_smallest_disk`        | Installs Flatcar Container Linux to the smallest disk.                                                                                                                                                                                                                                                                                                                                    | false                  | bool              | false    |
 | `install_disk`                    | Disk device where Flatcar Container Linux should be installed.                                                                                                                                                                                                                                                                                                                            | "/dev/sda"             | string            | false    |
-| `kernel_args`                     | Additional kernel args to provide at PXE boot.                                                                                                                                                                                                                                                                                                                                            | ["kvm-intel.nested=1"] | list(string)      | false    |
+| `kernel_args`                     | Additional kernel args to provide at PXE boot and in /usr/share/oem/grub.cfg.                                                                                                                                                                                                                                                                                                             | []                     | list(string)      | false    |
+| `kernel_console`                  | The kernel arguments to configure the console at PXE boot and in `/usr/share/oem/grub.cfg`.                                                                                                                                                                                                                                                                                               | ["console=tty0", "console=ttyS0"] | list(string) | false    |
```

Comment:
That would align it with the longer column width from `kernel_console`. `download_protocol` is also longer than the rest. Since the `kernel_args` line can be aligned with the rest, I would leave it as is.